catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -33,6 +33,15 @@
             description="Enables versioning of both metacards and content."/>
     </OCD>
 
+    <OCD name="Source Poller" id="ddf.catalog.util.impl.SourcePoller">
+        <AD name="Initial Delay" id="initialDelay" required="true" type="Integer"
+            default="0"
+            description="The Initial Delay (in seconds) before source polling starts."/>
+         <AD name="Period" id="period" required="true" type="Integer"
+            default="59"
+            description="The Period (or Interval) (in seconds) at which to execute successive source polling."/>
```

Comment:
I think the point of this change was to make it easier to make polling interval longer.  I would recommend using minutes instead of seconds.